x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts

```diff
@@ -165,112 +152,11 @@ export const getMetadataRequestHandler = function (
         ),
       });
     } catch (error) {
-      return errorHandler(logger, response, error);
+      return errorHandler(response, error);
```

Comment:
Yeah, the fact that the `errorHandler` was not calling `logger.*` is a bug IMO. That means that any `throw` exceptions that were  being caught in this code, were not being reported to the kibana log.

Reply:
done a4ef2a504cb68788255e6ae9d3d165701707c7b4